app/index.sass

```diff
@@ -30,6 +30,10 @@ body
 .container-narrow > hr
   margin: 30px 0
 
+svg
+  width: 100%
+  height: 100%
+
```

Comment:
This is fine for now - had to do this on my computer as well when I was playing with C3 this evening.  However, this is just a hack to get it to render, and we need to do the responsive stuff with `id=main-viz` anyway